package.json

```diff
@@ -19,7 +19,7 @@
     "db:migrate": "npm run knex migrate:latest",
     "db:seed": "npm run knex seed:run",
     "postinstall": "run-s db:migrate db:seed",
-    "test": "jest --watchAll"
+    "test": "jest -o --coverage"
```

Comment:
ooh take this out and you're good!

Reply:
@emilyparkes  hi sis all done from this commit  https://github.com/kahikatea-2019/MOA/pull/32/commits/b5f2d38f1cece24ebaf4a576677d021de90ef86d